### PR TITLE
setup.cfg: Add pytest opts

### PR DESCRIPTION
**Pull request by meetmangukiya**
Closes https://github.com/coala/corobo/issues/106

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

**Comment by meetmangukiya** [comment]
pending_review

**Comment by sils** [comment]
ack https://github.com/coala/corobo/pull/107/commits/c5daecfe734aee015b534f84fccd82df5707af67

**Comment by meetmangukiya** [comment]
thanks @sils

**Comment by meetmangukiya** [comment]
@rultor merge

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[tool:pytest]
+addopts =
+    tests
+    --cov plugins
+    --cov-fail-under=100
```
Comment (jayvdb) [inline setup.cfg]: These should be moved into `[coverage:run]` and `[coverage:report]` .  See coala/coala `setup.cfg`

**Pull request merged**